Improve benchmark running script (#2)
openfun/lbt: README.md

## Quick start guide (for developers)

Once you've cloned the project, start an LRS using:

```
$ make run-lrs-ralph
```

And then run a benchmark:

```
$ bin/bench \
      http://AAA:BBB@ralph:8090/xAPI \
      /app/locust/post.py \
      30 \
      1 \
      1
```

Usage of the `bin/bench` script is:

```
usage: bin/bench LRS_ROOT_URL PROTOCOL [DURATION] [CONCURRENT_USERS] [STATEMENTS_PER_REQUEST]
```

You will find your results in the latest runs directory:

```
$ ls runs | tail -n 1

> ```

Reviewer: Maybe we can wrap it in a make command `make results` ?

Author: :thinking: I like the idea of having a command / utility to list runs as a CSV rows:

```
$ bin/runs

users; duration; lrs_root_url; protocol; statements_per_request; run
1; 30; http://AAA:BBB@ralph:8090/xAPI/statements ; /app/locust/post.py; 1; 2023-03-31T17:00:25,608936440-04:00
1; 30; http://AAA:BBB@lrsql:8090/xAPI/statements ; /app/locust/post.py; 1; 2023-03-31T18:00:25,608926440-04:00
1; 30; http://AAA:BBB@learninglocker:8090/xAPI/statements ; /app/locust/post.py; 1; 2023-03-31T18:12:25,508936440-04:00
```

This allows cool rendering such as:

```
$ bin/runs | csvlook -I

| users |  duration |  lrs_root_url                                        |  protocol            |  statements_per_request |  run                                 |
| ----- | --------- | ---------------------------------------------------- | -------------------- | ----------------------- | ------------------------------------ |
| 1     |  30       |  http://AAA:BBB@ralph:8090/xAPI/statements           |  /app/locust/post.py |  1                      |  2023-03-31T17:00:25,608936440-04:00 |
| 1     |  30       |  http://AAA:BBB@lrsql:8090/xAPI/statements           |  /app/locust/post.py |  1                      |  2023-03-31T18:00:25,608926440-04:00 |
| 1     |  30       |  http://AAA:BBB@learninglocker:8090/xAPI/statements  |  /app/locust/post.py |  1                      |  2023-03-31T18:12:25,508936440-04:00 |
```

And we can also imagine filters such as:

```
$ bin/runs "ralph" | csvlook -I

| users |  duration |  lrs_root_url                                        |  protocol            |  statements_per_request |  run                                 |
| ----- | --------- | ---------------------------------------------------- | -------------------- | ----------------------- | ------------------------------------ |
| 1     |  30       |  http://AAA:BBB@ralph:8090/xAPI/statements           |  /app/locust/post.py |  1                      |  2023-03-31T17:00:25,608936440-04:00 |
```

Do you mind if we implement this in another PR?

You should see a bunch of CSV files created by `locust` and a `parameters.txt`

> files with used input parameters for this file:

Reviewer 2: ```suggestion
file containing the input parameters used for this run:
```
